ochap-backend/src/main/java/ch/heigvd/amt/backend/routes/ShoppingCartTemplate.java

```diff
@@ -0,0 +1,113 @@
+package ch.heigvd.amt.backend.routes;
+
+import ch.heigvd.amt.backend.entities.ProductQuantity;
+import ch.heigvd.amt.backend.entities.ShoppingCart;
+import ch.heigvd.amt.backend.repository.ProductQuantityDAO;
+import ch.heigvd.amt.backend.repository.ShoppingCartDAO;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import javax.persistence.EntityManager;
+import javax.validation.Valid;
+import java.util.List;
+import java.util.Optional;
+
+@Controller
+public class ShoppingCartTemplate {
```

Comment:
```suggestion
public class ShoppingCart {
```

Reply:
we already have a ShoppingCart class in the entities folder so I can't rename it that way